public/main.js

```diff
@@ -0,0 +1,246 @@
+/** Необходимые константы */
+const apiKey = "d1000af359abdc5da2cf58cb576e8c75";
+const getUrl = `http://ws.audioscrobbler.com/2.0/?method=tag.gettopalbums&tag=edm&api_key=d1000af359abdc5da2cf58cb576e8c75&format=json`;
+
+/** Вызов главного метода */
+getTop();
+
+/** Поисковая функция */
+document.querySelector(".header__search").addEventListener("keydown", (e) => {
+	if (e.key === "Enter") {
```

Comment:
Лучше смотреть на keyCode
```suggestion
	if (e.keyCode === 13) {
```